README.md

```diff
@@ -117,52 +117,60 @@ The Discord server is also the perfect place for sharing your feedback with us,
 
 ## Running the examples
 
-The readme for each example will explain how to get it running. Once the example is running, it needs to be discovered by an instance of the Restate runtime.
-
-### Launching the runtime
-
-Have a look at how to start up the runtime in a Docker container in [this repository]* or run the following commands:
-
-- For MacOS:
-```shell
-docker run --name restate_dev --rm -p 8080:8080 -p 9070:9070 -p 9071:9071 docker.io/restatedev/restate:latest
-```
-- For Linux:
-```shell
-docker run --name restate_dev --rm --network=host docker.io/restatedev/restate:latest
-```
-
-### Connect runtime and services
-
-Once the runtime is up, let it discover the services of the example by executing:
-
-- For MacOS:
-```shell
-curl -X POST http://localhost:9070/deployments -H 'content-type: application/json' -d '{"uri": "http://host.docker.internal:9080"}'
-```
-- For Linux:
-```shell
-curl -X POST http://localhost:9070/deployments -H 'content-type: application/json' -d '{"uri": "http://localhost:9080"}'
-```
+The readme for each example will explain how to get it running. Once the services of the example are running, the deployment needs to be registered in Restate.
+
+### Launching the Restate Server
+
+**NOTE:** Some examples can be run with Docker Compose. For those, you can ignore this section.
+
+For running Restate locally and downloading the binaries, have a look at the options on the [`Get Restate` page](https://restate.dev/get-restate/).
+
+- To run Restate in a Docker container:
+    ```shell
+    docker run --name restate_dev --rm -p 8080:8080 -p 9070:9070 -p 9071:9071 --add-host=host.docker.internal:host-gateway docker.io/restatedev/restate:latest
+    ```
+- To run Restate with `npx`:
+    ```shell
+    npx @restatedev/restate-server@latest
+    ```
+- To run Restate with Homebrew:
+    ```
+    brew install restatedev/tap/restate-server
```

Comment:
Should we also add the command for how to start the `restate-server`? Technically we are only installing the server via this command.